src/Employment/WorkContract_v1.0.py

```diff
@@ -0,0 +1,340 @@
+        title="Benefit Type",
+        description="The type of the taxable benefit",
+        examples=[BenefitType.PART_OF_SALARY, BenefitType.ADDITIONAL_TO_SALARY],
+    )
+    taxable_value: Optional[float] = Field(
+        None,
+        title="Taxable Value",
+        description="The taxable value for the compensation per month",
+        examples=[200],
+    )
+
+
+class DeterminationOfHoliday(str, Enum):
+    ANNUAL_HOLIDAY_ACT = "annual holiday act"
+    COLLECTIVE_AGREEMENT = "collective agreement"
+
+
+class Holidays(CamelCaseModel):
+    paid_holiday_included: bool = Field(
+        ...,
+        title="Paid Holiday",
+        description="Indicates if the contract includes paid holiday",
+        examples=[True],
+    )
+    number_of_holidays: Optional[int] = Field(
+        None,
+        title="Number of Holidays",
+        description="The terms used for the paid holidays",
+        examples=[5],
+    )
+    determination_of_holiday: Optional[DeterminationOfHoliday] = Field(
+        None,
+        title="Determination of Holiday",
+        description="The determination of the holiday",
+        examples=["The holiday is determined by the employer"],
+    )
+
+
+class Term(CamelCaseModel):
+    term_description: Optional[str] = Field(
+        None,
+        title="Term Description",
+        max_length=250,
+        description="The description of the term",
+        examples=[
+            "The employee is mandated to use workwear appointed by the Lapland Hotels and return them at the "
+            "end of the contract"
+        ],
+    )
+
+
+class WorkContractResponse(CamelCaseModel):
+    employer_info: EmployerInfo = Field(
+        ...,
+        title="Employer Info",
+        description="The details of the employer",
+    )
+    employee_info: EmployeeInfo = Field(
+        ...,
+        title="Employee Info",
+        description="The details of the employee",
+    )
+    terms_of_work: TermsOfWork = Field(
+        ...,
+        title="Terms of Work",
+        description="The terms and conditions for the work",
+    )
+    compensation: Compensation = Field(
+        ...,
+        title="Compensation",
+        description="The details of the compensation for the work",
+    )
+    holidays: Holidays = Field(
+        ...,
+        title="Holidays",
+        description="The details of the paid holidays",
+    )
+    benefits: Optional[List[Benefit]] = Field(
+        None,
+        title="Benefits",
+        description="The list of taxable benefits for the employee in addition to salary compensation",
+    )
+    termination: str = Field(
+        None,
+        title="Termination",
+        max_length=500,
+        description="The terms and conditions for the contract termination",
+        examples=[
+            "To terminate the work contract, the terminating party must provide 30 days of written notice to "
+            "the other party"
+        ],
+    )
+    other_terms: Optional[List[Term]] = Field(
```

Comment:
We've usually not made lists Optional, rather kept them required, but they can of course be empty lists. I updated the spreadsheet with Henna, so this field is now required (can be empty list).